include/boost/geometry/algorithms/detail/overlay/traversal.hpp

```diff
@@ -23,6 +23,7 @@
 #include <boost/range/value_type.hpp>
 
 #include <boost/geometry/algorithms/detail/overlay/cluster_info.hpp>
+#include <boost/geometry/algorithms/detail/overlay/debug_traverse.hpp>
```

Comment:
This file will be removed. But the debug info is used elsewhere, so it's moved out